lowstepper/src/lowstepper_rack/RackIORef.h

```diff
@@ -29,12 +29,13 @@ struct OutputRef {
     void setVolage(float voltage);
 };
 
-struct LightRef {
-    rack::engine::Light* light;
+struct GreenRedLightRef {
+    rack::engine::Light* redLight;
+    rack::engine::Light* greenLight;
 
-    LightRef() : light(nullptr) {}
-    float getBrightness();
-    void setBrightness(float brightness);
+    GreenRedLightRef() : redLight(nullptr), greenLight(nullptr) {}
```

Comment:
why does this initialize with a nullptr instead of void?

Reply:
Due to using some direct memory access for the vcv rack components, I think ensuring a default value for redLight and greenLight members of nullptr will catch any issues with uninitialization. It did catch me during development when I forgot to initialize greenLight for channel B once